### DISCUSSION TITLE
Pass in kedro mlflow arguments via command line interface

### DISCUSSION THREAD

**Discussion by luweilei304** [Galileo-Galilei/kedro-mlflow#414] [Q&A] [answered]
Hi team, 

I have defined a `conf/local/mlflow.yml` file. I am wondering if I can pass in the run name via kedro commend line. I tried `kedro run --params=tracking.run.name:"test_name"` but it didn’t work. Any ideas would be appreciated. Thanks!

**Comment by Galileo-Galilei** [comment, marked as answer]
Hi , actually this is not possible for the moment but there is an issue for this here: https://github.com/Galileo-Galilei/kedro-mlflow/issues/395. 

What is described in the issue (making configuraiton dynamic) is very hard, but I think I should accept passing kwargs at runtime as you suggest. Fell free to open a PR if you want to have this feature soon :)

**Reply by Galileo-Galilei** [reply]
This is now possible with the new OmegaConfigLoader, see above issue for reference!
